syntax = "proto3";

package presto;

service PrestoService {
    rpc ProcessPage(GrpcSerializedPage) returns (GrpcSerializedPage);
}

message GrpcSerializedPage {
    bytes sliceBytes = 1;
    sint64 positionCount = 2;
    sint64 uncompressedSizeInBytes = 3;
    bytes pageCodecMarkers = 4;
    sint64 checksum = 5;
}
